src/envs/pybullet_env.py

```diff
@@ -329,37 +329,81 @@ def _create_move_end_effector_to_pose_option(
         or closed according to the finger status.
         """
 
-        def _policy(state: State, memory: Dict, objects: Sequence[Object],
-                    params: Array) -> Action:
-            del memory  # unused
-            # First handle the main arm joints.
+        def _initiable(state: State, memory: Dict, objects: Sequence[Object],
+                       params: Array) -> bool:
+            # This is a really hacky hack to handle the situation where we are
+            # planning with a simplified model (that from the Blocks env) where
+            # states do not have joints. In this case, we want the option to
+            # be always initiable. Furthermore, we use pragma here because
+            # the oracle approach is not unit tested on this environment
```

Comment:
you don't need to run the whole oracle approach to unit test this. you can just create a PyBulletBlocks object, and grab an option using `env.options`, then run `option.initiable()`